pkg/pull/pull.go

```diff
@@ -221,6 +221,13 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 			return "", errors.Wrap(err, "failed to parse license from file")
 		}
 
+		if fetchOptions.License.Spec.ChannelID != airgap.Spec.ChannelID {
```

Comment:
Intentionally not checking for old style bundles with no channel ID because why?